src/views/pages/Home/index.jsx

```diff
@@ -1,20 +1,48 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import useModal from '../../../utils/useModal';
 import PageTitle from '../../components/PageTitle';
 import Button from '../../components/Button';
+import {Redirect} from 'react-router-dom';
+import {Link} from 'react-router-dom';
 
 function Home() {
+  const [state, setState] = useState({hasFennelAccount: false});
+
+  useEffect(() => {
+    //call to service to determine if they have a fennel account
+    //then set state
+    setState({hasFennelAccount: false});
+  }, []);
+
   const controls = useModal('Menu');
   const okdialog = useModal('OkDialog');
 
-  return (
-    <div className="flex flex-row">
-      <div className="basis-4/4 px-8">
-        <PageTitle>Home</PageTitle>
-        <Button onClick={() => controls.open()}>Open Menu</Button>
-        <Button onClick={() => okdialog.open()}>Ok Dialog</Button>
+  return !state.hasFennelAccount ? (
+    <>
+      <p>This page checks to see if there is evidence of an account.</p>
+      <p>
+        If no account then a bunch of introductory text explaining that acts as
+        a sort of splash followed by the option to getting started with creating
+        an account.
+      </p>
+      <p>
+        If there is an account, then we need to redirect to whatever page we
+        determine is approrpriate. It might be something like a dashboard of
+        some sort or maybe we just push to identities page and see if they have
+        an identity
+      </p>
+      <div className="mt-2">
+        <Link to="/account">
+          <Button type="button">Get Started</Button>
+        </Link>
```

Comment:
I believe the `<Link .. >` component has a property `renderAs` where you can specify to render it as a button instead of having to place a button inside the Link as a child

Reply:
removed due to page changes